### ISSUE TITLE
[Feature Request]: Make tubearchivist's playlist available in Jellyfin

### ISSUE THREAD

**Issue by Emporea** [tubearchivist/tubearchivist-jf-plugin#10]
### I've read the documentation

- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [feature request](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#feature-request) section.

### Your Feature Request

Hey, great plugin.
Are there some thoughts on how to implement the playlist feature in jellyfin via this plugin or by something else?
There are many things to consider that could make that work, but having seasons just by year makes it hard to follow a specific playlist created by a creator. It just skippes to the next video in time, when in fact the next video should be the one from the same playlist for example when watching a Let's Play.

### Your help is needed!

- [ ] Yes I will work on this in the next few days or weeks.

**Comment by DarkFighterLuke** [comment]
It should be feasible to synchronize TubeArchivist playlists with Jellyfin, but I need a deeper check.
Putting it to the backlog for the moment.

Thanks for your idea 👍🏽

**Comment by DarkFighterLuke** [comment]
The feature has been implemented with #68 and has been merged and released in v1.4.1.
